server/app.py

```diff
@@ -250,5 +250,6 @@ async def on_shutdown(app: web.Application):
     app.router.add_post("/offer", offer)
     app.router.add_post("/prompt", set_prompt)
     app.router.add_get("/", health)
```

Comment:
Should we remove the root path to `health`? since there's a new path?

Reply:
I don't think we can savely remove it without checking with the infra pod. They most likely use the main endpoint to check the health right now.